components/ai_chat/resources/page/state/conversation_context.tsx

```diff
@@ -464,12 +466,6 @@ export function ConversationContextProvider(props: React.PropsWithChildren) {
     })
   }
 
-  const resetSelectedSkill = () => {
```

Comment:
note: this (and the detectSkill from `input_box`) have been moved into an effect

Reply:
I see the `Reset` logic in the new `Effect`, but no `detect and select` logic. Do we no longer want to do this?